### PR TITLE
added dataset.json

### PR DESCRIPTION
**Pull request by SameerSahu007**
Hi I converted the titles list to dataset.json also ignore the previous pull request there was some changes which was not commited

### CODE REVIEW

**Review by jarmos-san — APPROVED** [review]
LGTM can merge @Bhupesh-V

File: dataset.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+               "doing code review", 
+               "deleting jira tickets", 
+               "Creating a pull request",
+               "forgot to run DB migrations", 
+               "Broke production", 
+               "running on staging ENV", 
+               "merging git branches", 
+               "releasing the MVP", "finding JIRA tickets"
+            ]
+}
```
Comment (Bhupesh-V) [inline dataset.json]: Hey its a nitpick but can you format the json file?
Use the following command to do so `python3 -m json.tool <filepath>`

File: main.py [hunk 2]
```diff
@@ -2,30 +2,18 @@
...
-    "Broke production",
-    "running on staging ENV",
-    "merging git branches",
-    "releasing the MVP",
-    "finding JIRA tickets"
-]
+with open('dataset.json', 'r') as f:
+    data = json.load(f)
+
+titles = data['titles']
```
Comment (Bhupesh-V) [inline main.py]: Please move this snippet inside `get_titles()` functions since get titles invloves reading the file as a subtask.

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]
Hey thanks for the contributions, have added some comments. Let me know if you have any questions

File: dataset.json (hunk 1 above)
Comment (jarmos-san) [reply dataset.json]: Reminds me, just FYI, I noticed some missing newlines at EOL as well which can introduce hard-to-debug line ending issues. I suggest configuring `git` to do it automatically for you whenever you commit changes. See [the docs](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings) for more information on the same.

cc @SameerSahu007

**Comment by SameerSahu007** [comment]
hi I made the changes you asked for

**Review by Bhupesh-V — APPROVED** [review]
LGTM @SameerSahu007 
Thanks for your contribution :+1:  :tada:

**Comment by Bhupesh-V** [comment]
@all-contributors please add @SameerSahu007 for code

**Comment by Bhupesh-V** [comment]
@all-contributors please add @SameerSahu007 for code

**Comment by Bhupesh-V** [comment]
@all-contributors please add @pavanjadhaw for ideas

**Pull request merged**